docs/UserGuide.md

```diff
@@ -91,6 +74,20 @@ Format: `profile -d INDEX`
 Example:
 * `profile -d 1` deletes the first profile listed.
 
+### View Profile: `profile -v`
+
+Shows either a list of profiles or a single profile.
+
+Format: `profile -v [INDEX]`
+
+Tip: `INDEX` is optional, specify to view single profile.
```

Comment:
Maybe having the view profiles before delete profiles make more sense, since the delete profile is based on the list shown.

Reply:
I agree, will move it up thanks.